docs/UserGuide.md

```diff
@@ -546,3 +546,15 @@ This section gives a quick summary of how you may use DonnaFin.
 | **Add Liability**   | `append n/LIABILITY_NAME ty/LIABILITY_TYPE v/$LIABILITY_VALUE r/REMARKS_ON_LIABILITY`| `append n/Property debt with DBS ty/debt v/$100000 r/10% annual interest`
 | **Add Policy**   | `append n/POLICY_NAME i/INSURER iv/$INSURED_VALUE pr/$YEARLY_PREMIUM c/$COMMISSION`| `append n/Diamond Policy i/AIA iv/$10000 pr/$200 c/$1000`
 | **Remove Asset/Liability/Policy**   | `remove INDEX`                                                                 | `remove 1`
+
+--------------------------------------------------------------------------------------------------------------------
+
+## 7. Glossary
+
+* **CLI**: Abbreviation for Command line interface, where commands are processed in the form of text.
+This is opposed to the standard graphical user interface that we are use to.
+* **Private client detail**: A client detail that is not meant to be shared with others
```

Comment:
I am not sure where we have used this. Maybe a more appropriate one would be client details, which we typically refer to as assets, liabilities,notes,policies,contacts. you get what i mean. pls don't quote me verbatim when u do the ug tho AHAHHAHA phrase it better

Reply:
we are **used to**

Reply:
ohh are you going to put the client details in?